### PR TITLE
Fix login redirection when browsing to /blog/create-article

### LINKED ISSUE #76: Cannot access article creation page by url
Title. Going to the link https://iscsc.fr/blog/create-article put the user back on the main blog page https://iscsc.fr/blog/

### PR DESCRIPTION
**Pull request by atxr**
Fix Login and SignUp redirection
Fix #76

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-  const { user } = useAuthContext();
+  const { user } = useAuthContext()
   return (
     <div className="App">
       <BrowserRouter>
         <Navbar />
         <div className="pages">
           <Routes>
             <Route path="/" element={<Navigate to="/blog" />} />
-            <Route path="/blog" element={<Blog />} />
```
Comment (ctmbl) [inline frontend/src/App.js:19]: This line deletion should be in a separated commit :wink: 
it's quite hard to find what is the "fix minor bug" in such a big formatting!

File: frontend/src/pages/Login.js [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
-  const { login, error, isLoading, ok } = useLogin();
-  const navigate = useNavigate();
-
-  const handleSubmit = async (e) => {
-    e.preventDefault();
-    await login(email, password);
+import { useNavigate } from 'react-router-dom'
+import { useAuthContext } from '../hooks/useAuthContext'
+import { useLogin } from '../hooks/useLogin'
+const { useState } = require('react')
```
Comment (ctmbl) [inline frontend/src/pages/Login.js:4]: If I'm not mistaken this again is formatting, not related to the next property adding (I reviewed commit by commit) !

**Review by ctmbl — CHANGES_REQUESTED** [review]
I must agree with @amtoine here, this would be waaaaay better if you could first create a PR that formats the whole repo, we would be in a much better state to review this one!

**Comment by atxr** [comment]
Waiting for #83

**Comment by atxr** [comment]
@ctmbl @amtoine 
Thanks to the CI, this PR is now ready!

**Review by amtoine — APPROVED** [review]
no new changes but a passing CI :ok_hand:

File: frontend/src/pages/Login.js (hunk 2 above)
Comment (amtoine) [reply frontend/src/pages/Login.js]: that is probably resolved @ctmbl, right? :yum:

File: frontend/src/App.js (hunk 1 above)
Comment (amtoine) [reply frontend/src/App.js:19]: is there something @atxr can do here?

Comment (atxr) [reply frontend/src/App.js:19]: I think it's to late, let's not waste time there.
Thanks to the ci it won't happen anymore ☺️

**Review by ctmbl — APPROVED** [review]

**Pull request merged**